demo/async-read-file.js

```diff
@@ -1,18 +1,30 @@
 // Run this with
 //
-//   traceur --experimental async-read-file.js file-to-read
+//   traceur --async-functions=true async-read-file.js
```

Comment:
Just ```--async-functions```does not work because async-functions has no default value. I got an error when I tried.

But ```--experimental``` works. I'm going to change.